File: PHP/Easy/Filters/Math/2413. Smallest Even Multiple.php
<?php

// https://leetcode.com/problems/smallest-even-multiple/
class Solution
{

    /**
     * @param Integer $n
     * @return Integer
     */
    function smallestEvenMultiple($n)
    {
        return $n % 2 === 0 ? $n : $n * 2;
    }
}

$obj = new Solution();
$result = $obj->smallestEvenMultiple(5);
echo $result;

File: Python/Easy/Filter/Math/2413. Smallest Even Multiple.py
def smallestEvenMultiple(n: int) -> int:
    # if n % 2 == 0:
    #     return n
    # else:
    #     return n * 2


    return n if n % 2 == 0 else n * 2


n = 6
print(smallestEvenMultiple(n))
